ql/src/test/results/clientpositive/llap/show_create_table_db_table.q.out

```diff
@@ -120,8 +120,8 @@ POSTHOOK: query: SHOW CREATE TABLE tmp_feng.tmp_showcrt4
 POSTHOOK: type: SHOW_CREATETABLE
 POSTHOOK: Input: tmp_feng@tmp_showcrt4
 CREATE TABLE `tmp_feng`.`tmp_showcrt4`(
-  `s1` struct<`p1`:string>, 
-  `s2` struct<`p2`:array<map<int,uniontype<struct<`a`:int, `b`:string>, array<struct<`c`:int, `d`:string>>>>>>)
+  `s1` struct<``p1``:string>, 
+  `s2` struct<``p2``:array<map<int,uniontype<struct<``a``:int, ``b``:string>, array<struct<``c``:int, ``d``:string>>>>>>)
```

Comment:
Hey @kasakrisz , what do you think about these changes? These occurred due to the change that you requested. I feel these are fine, but please let me know if you think otherwise.

Reply:
I tried to run this generated create table statement from q test but it failed.
The reason is that some struct fields names are escaped twice:
```
structElementName = HiveUtils.unparseIdentifier(structElementName, Hive.get().getConf()); // <- first
String structElementType = formatType(structTypeInfo.getAllStructFieldTypeInfos().get(i));

structFormattedType.append("`" + structElementName + "`:" + structElementType); // <- second
```

I also did some testing with struct fields having space and other special characters in their names. Unfortunately this is not supported so we can not create such tables.
I think the line
```
structElementName = HiveUtils.unparseIdentifier(structElementName, Hive.get().getConf());
```
should be reverted and for now.